### DISCUSSION TITLE
Migrating to React

### DISCUSSION THREAD

**Discussion by LassazVegaz** [pawan-live/timetable-app#87] [General]
# Migrating view to React

Hey guys! I am new here. Nice to meet you all
My good friend @pawan-live  joined me here...

I am starting off a new branch called `migrations/react` to migrate the project to [React](https://reactjs.org/).
If you have any ideas or contributing plans please comment

**Comment by LassazVegaz** [comment]
The new language will be [TypeScript](https://www.typescriptlang.org/)

**Comment by pawan-live** [comment]
Nice 🎉🎉

**Comment by pawan-live** [comment]
Will code the static UI part in JSX and update. And there will be additions to the UI ✨

**Reply by LassazVegaz** [reply]
Sure...😁
